sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveQl.scala

```diff
@@ -1307,16 +1307,9 @@ https://cwiki.apache.org/confluence/display/Hive/Enhanced+Aggregation%2C+Cube%2C
     HiveParser.DecimalLiteral)
 
   /* Case insensitive matches */
-  val ARRAY = "(?i)ARRAY".r
   val COALESCE = "(?i)COALESCE".r
```

Comment:
Why this isn't removed? I saw [`coalesce` included in #6710] [1].

[1]: https://github.com/apache/spark/pull/6710/files#diff-2c0350957ac4932d3f63796eceaeae08R89